### ISSUE TITLE
Breaking Change Not Documented

### ISSUE THREAD

**Issue by svozza** [tuplo/dynoexpr#50]
When I upgraded to v3.x from v2.x, I had to change my import to get it to work. This should be documented in the release.

```js
// Old
const dynoexpr = require('@tuplo/dynoexpr');

// New
const {default: dynoexpr} = require('@tuplo/dynoexpr');
```

**Comment by ruicsh** [comment]
hi @svozza that's a bug, you shouldn't need to do that. Can you tell me more about your setup please? node version, are you using typescript? If you could provide a demo repo, that would be great.

**Comment by ruicsh** [comment]
I got the error reproduced, you're using JS with CJS imports only

**Comment by ruicsh** [comment]
@svozza published v3.4.0, that fixes this problem. Let me know if there's any other issue.

**Comment by svozza** [comment]
Good stuff, it's working perfectly!
